skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+from scipy import ndimage
+
+
+def warp_images(
+    from_points,
+    to_points,
+    images,
+    output_region,
+    interpolation_order=1,
+    approximate_grid=2
+):
+    """Return an array of warped images.
+
+    Define a thin-plate-spline warping transform that warps from the
+    from_points to the to_points, and then warp the given images by
+    that transform.
+
+    Parameters
+    ----------
+    from_points: (N, 2) array_like
+        Source image coordinates.
+    to_points: (N, 2) array_like
+        Target image coordinates.
+    images: ndarray
+        Image to be warped with the given warp transform.
+    output_region: (1, 4) array
+        The (xmin, ymin, xmax, ymax) region of the output
+        image that should be produced. (Note: The region is inclusive, i.e.
+        xmin <= x <= xmax)
+    interpolation_order: int, optional
+        If value is 1, then use linear interpolation else use
+        nearest-neighbor interpolation.
+    approximate_grid: int, optional
+        If approximate_grid is greater than 1, say x, then the transform is
+        defined on a grid x times smaller than the output image region.
+        Then the transform is bilinearly interpolated to the larger region.
+        This is fairly accurate for values up to 10 or so.
+
+    Returns
+    -------
+    warped: array_like
+        Array of warped images.
+
+    References
+    ----------
+    .. [1] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
```

Comment:
From the update in the LICENSE file, I would expect a reference to the 2007 work by Zachary Pincus under either References or Notes.

Reply:
Doesn't hurt and certainly feels polite. :)